mm2src/mm2_main/src/lp_wallet.rs

```diff
@@ -550,3 +525,38 @@ pub async fn get_wallet_names_rpc(ctx: MmArc, _req: Json) -> MmResult<GetWalletN
         activated_wallet: activated_wallet.clone(),
     })
 }
+
+/// `ChangeMnemonicPasswordReq ` represents a request to update
+/// the password for the seed storage.
```

Comment:
Mnemonic not seed :)

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2317/commits/d9647de56092433b79756a9a29915dded5180efe